network-api/networkapi/buyersguide/templates/about/press.html

```diff
@@ -9,49 +9,183 @@ <h1 class="h2-heading">
       {% trans "Press" %}
     </h1>
 
+    <p>
+      {% blocktrans trimmed %}
+      <em>*Privacy Not Included</em> helps consumers shop for trustworthy connected products. But it also helps journalists
+      cover connected products. Our research assists reporters in incorporating product privacy &mdash; and not just price
```

Comment:
@TheoChevalier do we prefer `&mdash;` or do we want `—`?

Reply:
This was resolved in slack. 👍